content/en/docs/High Availability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "High Availability"
+linkTitle: "High Availability"
+weight: 170
```

Comment:
I suggest weighing this so it's before "Releases". In the order of doc links on the side bar, "Releases" is typically last in most projects.